packages/table/src/layers/guides.tsx

```diff
@@ -24,6 +24,11 @@ export interface IGuideLayerProps extends IProps {
 }
 
 export class GuideLayer extends React.Component<IGuideLayerProps, {}> {
+
+    // this should be consistent with the guide width/height specified in CSS.
+    // TODO: find a way to determine this programmatically - maybe after render?
```

Comment:
two options:
1. set the size of the guide element here in JS (and _not_ in CSS, so it's only set in one place), or
2. refactor CSS such that the logic here does not need to account for dimensions of guide in the DOM. like can the offset to center the guide be divorced from its positioning somehow? perhaps by using `transform` instead of negative margin in the CSS?